modules/ROOT/nav.adoc

```diff
@@ -5,7 +5,12 @@
 ** xref:get-started:architecture.adoc[How Redpanda Works]
 ** xref:get-started:install-beta.adoc[Install Beta]
 ** xref:get-started:quick-start.adoc[Quickstart]
-** xref:get-started:licenses.adoc[Redpanda Licensing]
+** xref:get-started:licensing/index.adoc[Redpanda Licensing]
+*** xref:get-started:licensing/overview.adoc[Editions and Enterprise Features]
+*** xref:get-started:licensing/add-license-redpanda/index.adoc[Add a License]
```

Comment:
Should Enterprise appear in the label just in case users are scanning for this rather than reading?